ush/templates/FV3LAM_wflow.xml

```diff
@@ -568,6 +568,8 @@ MODULES_RUN_TASK_FP script.
 {% else %}
            <and>
              <timedep><cyclestr offset="&START_TIME_CONVENTIONAL;">@Y@m@d@H@M00</cyclestr></timedep>
+             <taskdep task="&MAKE_ICS_TN;"/>
+             <taskdep task="&MAKE_LBCS_TN;" cycle_offset="-{{ extrn_mdl_ics_offset_hrs }}:00:00"/>
```

Comment:
Why we need these two dependencies now?

Reply:
As we discussed in today's meeting, we started to try to make more boundary dependence so we know which boundary  files to use. This one is only to prevent cold start to start too early without boundary task done.